### DISCUSSION TITLE
Typo in Unit 3.5 Quiz

### DISCUSSION THREAD

**Discussion by harishb00** [Lightning-AI/dl-fundamentals#94] [Lecture videos and quizzes] [answered]
![image](https://github.com/Lightning-AI/dl-fundamentals/assets/43300971/633d89dd-328f-42bf-9cf8-d644ede77e39)

It should be **are**, I think. It's silly to mention this as a typo. But the course is really great and @rasbt  and the Lightning AI team is doing a fantastic job. So trying to make small contributions to perfect the already the perfected course. :)

**Comment by rasbt** [comment, marked as answer]
Thanks for the nice words, I am really glad that you are liking the course. Also, thanks for pointing out the typo, just fixed it!
